java/src/com.zeroc.ice/src/main/java/com/zeroc/Ice/ObjectPrx.java

```diff
@@ -211,108 +210,108 @@ CompletableFuture<Object.Ice_invokeResult> ice_invokeAsync(
             Map<String, String> context);
 
     /**
-     * Returns the identity embedded in this proxy.
+     * Gets the identity embedded in this proxy.
      *
      * @return The identity of the target object.
      */
     Identity ice_getIdentity();
 
     /**
-     * Returns a proxy that is identical to this proxy, except for the identity.
+     * Creates a proxy that is identical to this proxy, except for the identity.
```

Comment:
Every other file was updated to say `Creates a new`, instead of just `Creates`.
```suggestion
     * Creates a new proxy that is identical to this proxy, except for the identity.
```
But for some reason, this file's comments omit the `new` in many places.